cmd/blockscopy/main.go

```diff
@@ -50,7 +52,9 @@ type config struct {
 func (c *config) RegisterFlags(f *flag.FlagSet) {
 	f.StringVar(&c.sourceBucket, "source-bucket", "", "Source GCS bucket with blocks.")
 	f.StringVar(&c.destBucket, "destination-bucket", "", "Destination GCS bucket with blocks.")
-	f.DurationVar(&c.minBlockDuration, "min-block-duration", 24*time.Hour, "If non-zero, ignore blocks that cover block range smaller than this.")
+	f.DurationVar(&c.minBlockDuration, "min-block-duration", 0, "If non-zero, ignore blocks that cover block range smaller than this.")
+	f.Var(&c.minTime, "min-time", "If set, only blocks with MinTime >= this value are copied.")
```

Comment:
Can you add an example of the expected format for these?

Reply:
Good idea. Done in: https://github.com/grafana/cortex-tools/pull/261/commits/ce4619c3ea97fa7f5d135ef2a597cf902c73fdfc